benchmark/single_doc/run_command.hpp

```diff
@@ -45,7 +45,7 @@ class run_command : public microbench {
     mongocxx::database _db;
 };
 void run_command::task() {
-    auto command = make_document(kvp("ismaster", true));
+    auto command = make_document(kvp("ping", 1));
```

Comment:
```suggestion
    auto command = make_document(kvp("hello", true));
```

This comes from the [Benchmarking spec](https://github.com/mongodb/specifications/blob/master/source/benchmarking/benchmarking.rst#run-command).

Each benchmark defines a `Dataset size` for measurement. Please update the `0.16` above to `0.13`. I believe `0.16` corresponds to the [previous size](https://github.com/mongodb/specifications/commit/99c9ee188a4bc4fc095837ae0be2451c30761c9b#diff-289c90aed81a3162c93a75dc51bf0f7815a4de7e99283a1007e0624d91c06004R492) of 160,000 bytes.

Reply:
Ah, of course.

Reply:
Also update the `0.16` value on L34 to `0.13`.